docs/project-overview.md

```diff
@@ -0,0 +1,83 @@
+* The Policy can access Reference Values or Endorsements required to support the Appraisal by specifying queries
+* The API will record basic metrics for integration into deployment operational schemes
+* In addition to the direct Verification of the supplied evidence, further policy can be deployed to add 'Derived Claims' to the Attestation results.
+
+### Scope - Provisioning
+Some of the more complex problems in the implementation of Verification Services arise from the requirements of Appraisal Policy to access Reference Values for Evidence. These values must be supplied by a trusted supply chain and kept securely by the Service. The Veraison project supports a Provisioning module that allows Reference Values (& Endorsements) to be presented by the supply chain and stored within the service. The actual storage medium is abstracted from the storage model. It has been determined that the most appropriate form of storage to support queries by the validator is a database that support graph structures & queries.
+Given the above, the scope of the Provisioning work is:
+* API to allow provisioning of Reference Values / Endorsements from supply chain. Secured provisioning actions will be validated.
+* Bulk provisioning
+* Upgrade Relationships for Firmware components
+* Targeted Object Revocation
+* Abstracted Storage
+* Mapped to underlying Data Model to support Verification operations
+* Support for some classes of remotely stored Reference Values (e.g. Project Trillian based Firmware Transparency)
+* Multi Tenancy data separation
+* AuthN & AuthZ access control for API operations & data separation
+* Audit Trail for Provisioning operations
+* Potential to extend operations for multi region support
+
+### Scope - Reference Deployments
+Reference deployments show that the APIs can be practically applied. The intent is that the Reference Deployments implemented can be a starting point for production deployments.
+* Self Hosted deployment e.g. servers in a corporate data centre
+* PAAS Hosted deployment e.g. AWS / Azure / GCP (other hyperscalers are available)
+* APIs will be hosted and externally available
+* Storage will be provided to support the data model
+* Scaling out of API compute units in response to load
+* Simple AuthN schemes
+* Metric & Audit storage
+
+The following are currently out of scope for the Reference Deployments:
+* Full devops / CD work
+* SLA monitoring
+* Billing
+
+
+## What's out of Scope for the Project
+It is not intended to look at other aspects of Attestation e.g.
+* Unification of Attestation Token formats
+* Normalising Relying Party Attestation request API
+* Common Attestation protocols
+
+
+## Programming Language
+GoLang is used as the implementation language for the project. This choice reflected the need for an efficient, memory managed language with good interoperability to other systems.
```

Comment:
I would add a note about the plugin system being language-agnostic.  It seems like a plus point especially for integrators that have already code (or expertise) in a certain language != go.

Reply:
Good point